specification/repository.md

```diff
@@ -144,3 +144,9 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 - MUST document how to configure log correlation
 - MUST define minimum supported version of each auto-instrumentation framework
   - SHOULD define maximum supported version of each auto-instrumentation framework
+
+## Real User Monitoring Libraries
+
+- MUST document all supported configuration parameters and how to set them
```

Comment:
Technicality, but, this is a bit awkward for otel-js-browser, because all ENV variables supported by Open Telemetry package itself can be set on `window` and they'll get picked up.

Reply:
I think that you can just link OTel docs and say "we support all of these too". That's what we do in instrumentation distros

Reply:
What I mean is that I'm not sure we (Browser RUM) support all OTel config params / ENV variables right now in a sane way, ie. if there's an OTel option which messess up something in our configuration.

Reply:
Oh, I see. Then I would consider all options not described in the docs as unsupported; you could even add a warning "don't use OTel config directly, this project wraps over OTel and configuring the lower level SDK directly may break something"